Components/ArticleList.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from "react";
+import { fetchArticles } from '../utils'
+import { Link } from 'react-router-dom';
+
+
+function ArticleList() {
+    const [articles, setArticles] = useState([]);
+
+    useEffect(() => {
+        fetchArticles().then(({articles}) => {
+            setArticles(articles);
+        })
+    }, []);
+
+
+
```

Comment:
Please set up a state for loading and use conditional rendering to let the users know, that the articles are still loading.